addons/goldpinger/template/testgrid/tests.yaml

```diff
@@ -55,6 +55,14 @@
     # print goldpinger output (and fail if unable to connect to the service)
     curl $GP_ENDPOINT/check_all
     curl $GP_ENDPOINT/metrics
+    
+    # For migration tests - check if this is the Bloomberg version
+    if kubectl get daemonset -n kurl goldpinger -o jsonpath='{.spec.template.spec.containers[0].image}' | grep "bloomberg"; then
```

Comment:
both charts use the bloomberg image `image: "bloomberg/goldpinger:3.10.0"`

Reply:
yeah, i just realized this as well when i was testing thanks for the call out